constellations/gemini-user/src/internship/internship.js

```diff
@@ -0,0 +1,116 @@
+                <p>
+                    Math Navigator occassionally offers Software Development
+                    Internship for ambitious students who would like to pursuse
+                    a college major or a career in software development. In this
+                    internship oppotunity, students will learn the fundamentals
+                    of modern technologies commonly used across
+                    engineering-focused companies. And with these skills,
+                    students help create this site that you are currently using!
+                </p>
+                <h1>Technology Stack</h1>
+
+                <div class="container-main">
+                    <div class="child-image">
+                        <img class="image" src={ApCalcImgScr} />
+                    </div>
+                    <div class="child-caption">
+                        <p>
+                            Math Navigator occassionally offers Software
+                            Development Internship for ambitious students who
+                            would like to pursuse a college major or a career in
+                            software development.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="container-main">
+                    <div class="child-image">
+                        <img class="image" src={ApCalcImgScr} />
+                    </div>
+                    <div class="child-caption">
+                        <p>
+                            Math Navigator occassionally offers Software
+                            Development Internship for ambitious students who
+                            would like to pursuse a college major or a career in
+                            software development.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="container-main">
+                    <div class="child-image">
+                        <img class="image" src={ApCalcImgScr} />
+                    </div>
+                    <div class="child-caption">
+                        <p>
+                            Internship opportunities will be announced when they
+                            are available. When they are, student candidates
+                            must first pass a series of coding assessments and
+                            interviews in order to receive a position. This
+                            interview process reflects the same process that
+                            famous technology companies like Google, Facebook,
+                            Amazon, etc. have. We encourage all students to
+                            attempt these assessments to familiarize themselves
+                            with this interview process structure if they ever
+                            want to pursue a career at any of these companies.
```

Comment:
I'm not sure if this would work... but maybe you could give this specific caption and image different class names so that you could assign separate css to make it fit better? Or maybe @ahsu1230 has a way to modify your current css so that it fits nicely.

Reply:
since these images and captions are just placeholders, i wanted to style it in a way that no matter what we insert into the placeholders, the captions will be aligned and the images will be the same size and aligned. i thought about styling each container separately but if we insert something other than the placeholder, then it'll probably look messy again. (fyi i gave the container and the images border colors so it's easier to see the container height etc)

Reply:
Good point cathy. For now, I this is fine because I don't have real text yet. They're not going to be big paragraphs anyway. Probably a sentence or two.